contestantB/src/main/java/edu/asu/diging/contestantB/config/KafkaTopicListener.java

```diff
@@ -0,0 +1,45 @@
+package edu.asu.diging.contestantB.config;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.annotation.PostConstruct;
+
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.serialization.IntegerSerializer;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.kafka.core.DefaultKafkaProducerFactory;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.kafka.core.ProducerFactory;
+
+
+public class KafkaTopicListener {
+	private org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(getClass());
+	@KafkaListener(id = "test.listener.id", topics = "my_topic")
```

Comment:
should be a real listener id and a real topic (I think you decided on a topic name?)